src/ts/UIFactory.ts

```diff
@@ -345,9 +345,9 @@ function uiLayout(config: UIConfig) {
 }
 
 function adsUILayout() {
-  let controlBar = new ControlBar({
+  let controlBar = new AdControlBar({
     components: [
-      new Container({
+      new Container(<ContainerConfig>{
```

Comment:
I think we can remove the type cast here


```suggestion
      new Container({
```